extensions/amp-story/1.0/amp-story.js

```diff
@@ -328,6 +322,14 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {?number} the timeout to show subscriptions dialog after delay */
     this.showSubscriptionsUITimeout_ = null;
+
+    /** @private {?number} the index of the page where the paywall would be triggered. */
+    this.subscriptionsPageIndex_ = this.storeService_.get(
+      StateProperty.SUBSCRIPTIONS_PAGE_INDEX
+    );
+
+    /** @private {!Deferred} a promise that is resolved once the subscriptions page index is extracted from amp-story-subscriptions. */
+    this.subscriptionsPageIndexPromise_ = new Deferred();
```

Comment:
The variable is called `Promise` but is of type `Deferred`, it would make more sense to be called `this.subscriptionsPageIndexDeferred_`